spec/acceptance/foreman_cli_plugins_spec.rb

```diff
@@ -17,15 +17,17 @@ class { 'foreman::cli':
         if $facts['os']['family'] == 'RedHat' {
           include foreman::cli::ansible
           include foreman::cli::azure
+          include foreman::cli::kubevirt
```

Comment:
Line 37+ then also needs updating? ;)

Reply:
I now added tests for the RH-only plugins too.